cmd/jimmsrv/main.go

```diff
@@ -45,6 +45,7 @@ func start(ctx context.Context, s *service.Service) error {
 	logLevel := os.Getenv("JIMM_LOG_LEVEL")
 	logDevMode, _ := strconv.ParseBool(os.Getenv("JIMM_LOG_DEV_MODE"))
 	logger.SetupLogger(ctx, logLevel, logDevMode)
+	logger.LogJimmStartup(ctx)
 	zapctx.Info(ctx, "jimm info",
```

Comment:
Can you move this `jimm info` into LogJimmStartup please, seems like a good place for it.

Reply:
Ah but I see it's just for a single event so nvm this then.